pkg/thumbnails/storage/storage.go

```diff
@@ -1,11 +1,12 @@
 package storage
 
+import "github.com/owncloud/ocis-thumbnails/pkg/thumbnails/resolution"
+
 // Context combines different attributes needed for storage operations.
 type Context struct {
```

Comment:
Have you give it a thought about using golang's context api? :D

Reply:
Yes, I did but isn't golangs context api a bit overkill for our use case?
And also a bit too generic for a method api?

Reply:
Context is not overkill, it's the simplest package out there :P

Also:

>  Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx

with emphasis on ctx as first argument.

I think what you're referring to as context, as you said, can be something else.

The pros of using `context.Context` is once you adopt the idiom, goroutines get way, way easier; if you intend using goroutines here anyway.

I think renaming it to something else would be less confusing :)